obs_github_deployments.gemspec

```diff
@@ -28,8 +30,9 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{\Aexe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  # Uncomment to register a new dependency of your gem
-  # spec.add_dependency "example-gem", "~> 1.0"
+  spec.add_development_dependency "rake", "~> 13.0"
```

Comment:
LGTM, I just wonder if we should use `add_development_dependency` or  `add_dependency` for `rake` :thinking:

Reply:
I don't have a use case in mind where you would need rake outside of development, at least at the current state

Reply:
Yes, you are right.